fighthealthinsurance/models.py

```
@@ -825,3 +828,128 @@ class StripeWebhookEvents(models.Model):
+    # Timestamps
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    @classmethod
+    def filter_to_allowed_requests(cls, current_user):
+        """Filter to requests that the current user is allowed to see."""
+        if not current_user.is_authenticated:
+            return cls.objects.none()
+
+        try:
+            professional_user = get_object_or_404(ProfessionalUser, user=current_user)
```

Ruff (0.8.2)

`ProfessionalUser` may be undefined, or defined from star imports

(F405)